subsystems/intakesubsystem.py

```diff
@@ -161,13 +160,15 @@ def periodic(self) -> None:
                 NEOBrushless.ControlMode.Velocity, constants.kIntakeSpeed * -1
             )
 
+        self.hasPosition = backLimitState or frontLimitState
```

Comment:
why front or back? before with only back sensor meant it would only keep the position if the back sensor was covered, now it would keep the position when the front sensor is covered (not fully in the intake)

Reply:
todo later: have it stop if triggered while intaking, then while in holding position update to center the note (which would sorta be while moving toward transfer)